src/components/dashboard/FirstSteps/index.tsx

```diff
@@ -0,0 +1,90 @@
+import useBalances from '@/hooks/useBalances'
+import { useAppSelector } from '@/store'
+import { selectOutgoingTransactions } from '@/store/txHistorySlice'
+import { type ReactNode, useMemo } from 'react'
+import { Card, WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { LinearProgress, List, ListItem, ListItemIcon, Typography } from '@mui/material'
+import CircleOutlinedIcon from '@mui/icons-material/CircleOutlined'
+import CheckCircleRoundedIcon from '@mui/icons-material/CheckCircleRounded'
+
+const StatusItem = ({ children, completed = false }: { children: ReactNode; completed?: boolean }) => {
+  return (
+    <ListItem disableGutters>
+      <ListItemIcon sx={{ minWidth: 24, mr: 1, color: (theme) => theme.palette.primary.main }}>
+        {completed ? <CheckCircleRoundedIcon /> : <CircleOutlinedIcon />}
+      </ListItemIcon>
+      {children}
+    </ListItem>
+  )
+}
+
+type StatusProgressItems = Array<{
+  name: string
+  completed: boolean
+}>
+
+const calculateProgress = (items: StatusProgressItems) => {
+  const totalNumberOfItems = items.length
+  const completedItems = items.filter((item) => item.completed)
+  return Math.round((completedItems.length / totalNumberOfItems) * 100)
+}
+
+const StatusProgress = ({ items, completedName }: { items: StatusProgressItems; completedName?: string }) => {
+  const progress = calculateProgress(items)
+
+  return (
+    <>
+      <List disablePadding sx={{ py: 3 }}>
+        {items.map(({ name, completed }) => {
+          return (
+            <StatusItem key={name} completed={completed}>
+              {name}
+            </StatusItem>
+          )
+        })}
+        {completedName && <StatusItem completed={progress === 100}>{completedName}</StatusItem>}
```

Comment:
Aren't all items supposed to be shown at all times? And if not completed it's grey?